### PR TITLE
fix(Operating Account)

### LINKED ISSUE #3601: (bug) Incorrect calculation in the operating account report
Calculation and display errors were unfortunately introduced in the report of the operating account

### PR DESCRIPTION
**Pull request by lomamech**
- Update request fot getting Account Balances By Type Id, get the amount
  by debit - credit instead credit - debit
- Multiply Incomes and Total Income by (-1) for not have to display a
  negative value

closes #3601

### CODE REVIEW

**Comment by lomamech** [comment]
[Operating Account.pdf](https://github.com/IMA-WorldHealth/bhima/files/2875475/Operating.Account.pdf)

**Comment by lomamech** [comment]
@jniles  @mbayopanda  I finish, I need a review

File: server/controllers/finance/fiscal.js [hunk 1]
```diff
@@ -738,12 +738,13 @@ function getPeriods(req, res, next) {
 
 /**
  * return a query for retrieving account'balance by type_id and periods
+ * In general in accounting the balance is obtained by making debit - credit
  */
 function getAccountBalancesByTypeId(rate = 1) {
   return `
     SELECT ac.id, ac.number, ac.label, ac.parent, (IFNULL(s.amount, 0)*${rate}) AS amount, s.type_id
     FROM account as ac LEFT JOIN (
-    SELECT SUM(pt.credit - pt.debit) as amount, pt.account_id, act.id as type_id
```
Comment (jniles) [inline server/controllers/finance/fiscal.js:746]: Is this function used anywhere else?  We should make sure that this change doesn't affect any other reports.

Comment (lomamech) [reply server/controllers/finance/fiscal.js:746]: I made a request and I notice it only used in the Operating Account report \server\controllers\finance\reports\operating

Comment (jniles) [reply server/controllers/finance/fiscal.js:746]: Super.  Looks good to me

**Pull request merged**